tests/eval/test_2fa_duo.py

```diff
@@ -0,0 +1,408 @@
+"""
+This set of tests is based on the manual tests found in the
+IdP Release Candidate Test Matrix Version 4.2.4 (5/27/2021) here
+https://wiki.cac.washington.edu/display/SMW/IAM+Team+Wiki
+
+2FA-1 thru 2FA-11. 2FA-8b and 2FA-10 are not yet automate-able.
+"""
+
+import pytest
+
+from webdriver_recorder.browser import Chrome
+
+from tests.helpers import CTRL_KEY, Locators, WebTestUtils
+from tests.models import ServiceProviderInstance, AccountNetid, TestSecrets
+
+from selenium import webdriver
+
+
+@pytest.fixture
+def browser() -> Chrome:
+    return Chrome()
+
+
+@pytest.fixture(scope='class')
+def class_browser() -> Chrome:
+    browser = Chrome()
+    try:
+        yield browser
+    finally:  # Even if there is an error
+        browser.close()
+
+
+@pytest.fixture
+def new_tab(browser):
+    def inner():
+        browser.execute_script("window.open('');")
+        browser.switch_to.window(browser.window_handles[-1])
+    return inner
+
+
+@pytest.fixture
+def close_tab(browser):
+    def inner():
+        browser.find_element_by_tag_name('body').send_keys(CTRL_KEY + 'w')
+    return inner
```

Comment:
These 4 fixtures should be placed in one location and one location only: The root `conftest.py`

All other tests in the suite will be able to use them thereafter.